examples/pubspec.yaml

```diff
@@ -0,0 +1,49 @@
+name: flame_example
+description: A set of small game examples showcasing each feature provided by the Flame Engine.
+publish_to: 'none'
+
+version: 0.1.0-rc2
+
+environment:
+  sdk: ">=2.10.0 <3.0.0"
+
+dependencies:
+  flame:
```

Comment:
This should be a relative path now right?b

Reply:
yes